server/table.go

```diff
@@ -121,6 +124,20 @@ func (t *Table) getColumnsByName(names []string) ([]*Column, error) {
 	return columns, nil
 }
 
+func (t *Table) addForeignKeyConstraint(stmt *ast.CreateTable, parentStmt *ast.CreateTable) {
```

Comment:
better to move this logic into `database.CreateTable` because query to sqlite is built in database methods